source/_components/noaaweather.markdown

```diff
@@ -0,0 +1,116 @@
+---
+layout: page
+title: "NOAA/NWS Sensor"
+description: "How to integrate NOAA/National Weather Service observations within Home Assistant."
+date: 2019-04-15 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: 
+ha_category:
+  - Sensor
+ha_release: 0.94
+ha_iot_class: Cloud Polling
+---
+
+The `noaaweather` platform uses the [US NOAA/National Weather Service](https://www.weather.gov/) web service as a source for meteorological data for your location. The location is based on the `longitude` and `latitude` coordinates configured in your `configuration.yaml` file. The specific observation station for that area can be specified, or the closest station will be used. A majority of the observation stations are located at airports, and identified by the ICAO airport code.
+
+## {% linkable_title Configuration %}
+
+To add NOAA/NWS Weather to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: noaaweather
+    useragent:	email@example.com
```

Comment:
I think that we can remove the `useragent`, only required items in the first example.